conf/core/ConfigOptions.asciidoc

```diff
@@ -841,6 +841,14 @@ performing a bounded API database query but are immediately connected to a way t
 crosses the bounds are tagged with a custom tag. Must be used in conjunction with
 `convert.bounding.box.keep.immediately.connected.ways.outside.bounds`. Not supported by all readers.
 
+=== convert.require.area.for.polygon
```

Comment:
This is explained some in `ElementConverter` but basically, we require an element to support an area geometry in the schema in order for it to be turned into a GEOS polygon. Unfortunately, this doesn't work for polygon elements not defined in the schema. I tried removing the requirement and got some really strange test output changes on a few tests. So, instead decided to make the behavior configurable and the only time this gets set to false is during calls from some of the new generic geometry conflation scripts.

Reply:
I think we might need to revisit the internal schame. I don't think it has been updated in a fair while. Adding features might help.

Reply:
Right. I'd like to run the scraping script again soon...its been years (#3187) But we'll never probably be able to account for every feature that should have an area geometry and generic polygon conflation is meant to catch all the polys we don't describe in the schema...so that's basically what this option allows for.

Reply:
Wow!
Keep seeing email notifications and `convert.bounding.box.keep.immediately.connected.ways.outside.bounds` caught my eye.  That's I'm trying to do in services code so that the hoot reference dataset has the features it needs to make sure cut & replace ways get snapped.

Reply:
Had to refresh on what that's actually doing... changeset replacement auto sets it to true for all linear features if the lenient bounds option is enabled. When enabled, during data reading it will tag all the ways connected one step out to the in bounds ways. Then, when the changeset gets created it won't allow any ways with the corresponding tags to be deleted by the changeset.